src/components/Editor/index.js

```diff
@@ -280,20 +280,28 @@ const Editor = React.createClass({
     const { selectedToken } = this.state;
     const cm = this.editor.codeMirror;
     const token = e.target;
+    const tokenText = token.innerText;
 
-    if (!selectedFrame || !selectedToken ||
-        !sourceText || !isEnabled("editorPreview")) {
+    if (!selectedFrame || !sourceText || !isEnabled("editorPreview")) {
       return;
     }
 
-    selectedToken.classList.remove("selected-token");
+    if (selectedToken) {
+      selectedToken.classList.remove("selected-token");
+    }
```

Comment:
adding `this.setState({ selectedToken: null });` above this line fixes the Popover not getting removed for the previous element.